pkg/landscaper/execution/classification.go

```diff
@@ -140,7 +140,7 @@ func newDeployItemClassificationForDelete(executionJobID string, items []*execut
 			if item.DeployItem.Status.GetJobID() != item.DeployItem.Status.JobIDFinished {
 				c.runningItems = append(c.runningItems, item)
 			} else if item.DeployItem.Status.GetJobID() == item.DeployItem.Status.JobIDFinished &&
-				item.DeployItem.Status.DeployerPhase != lsv1alpha1.DeployerPhases.Failed {
+				item.DeployItem.Status.Phase != lsv1alpha1.DeployItemPhases.Failed {
```

Comment:
If we distinguish between phases `Failed` and `DeleteFailed`, then here it should be `DeleteFailed`. We could also use the `IsFailed` method.

Reply:
That's what the `IsFailed` function is for. I just missed that line. Will fix it.